src/main/java/uk/gov/companieshouse/lookup/controller/CompanyConfirmationController.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.companieshouse.lookup.controller;
+
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+
+@Controller
+@RequestMapping("/accounts/confirmation")
```

Comment:
Should this route be `/accounts/confirmation`? What if this was used outside of accounts?